backend/app/services/channels/telegram/emitter.py

```
@@ -0,0 +1,345 @@
+                )
+
+                await self._bot.edit_message_text(
+                    chat_id=self._chat_id,
+                    message_id=self._message_id,
+                    text=display_content,
+                )
+
+                self._pending_content = ""
+                self._last_update_time = current_time
+
+            except Exception as e:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)